src/utils/jwt/custom-claims.ts

```diff
@@ -108,7 +108,11 @@ export const generateCustomClaims = async (userId: string) => {
         // * Parse the path into a JSONata AST
         const expression = jsonata(path);
         // * Evaluate the JSONata expression from the fetched user data
-        const value = expression.evaluate(user, expression);
+        let value = expression.evaluate(user, expression);
+        // * If the claim is expected to be an array and its value is undefined, set its value to an empty array
+        if (value == undefined && path.includes('[]')) {
```

Comment:
put a `===` instead of a `==` to make a strict type comparison

Reply:
Not sure about that, they wrote `if (value != undefined)` a few lines below. I wanted to do equivalent check because I actually want to do the same if the value is `null` for example.

Reply:
Yes but you rather be explicit about it, `value === undefined || value === null`, explicit better than implicit.
It's even an eslint rule because it's very common https://eslint.org/docs/latest/rules/eqeqeq